components/copy_button.tsx

```diff
@@ -31,11 +32,25 @@ const CopyButton: React.FC<Props> = (props: Props) => {
         copyToClipboard(props.content);
     };
 
+    const getId = () => {
+        if (isCopied) {
+            return t('copied.message');
+        }
+        return props.beforeCopyText ? t('copy.text.message') : t('copy.code.message');
```

Comment:
This change also changes the message from `Copy code block` to `Copy code` if a piece of code is directly written in channel using ``` (check the screenshot below).

I suppose this change is okay to be consistent across different pages when using the same button to do the same action. 

<img width="577" alt="Screen Shot 2022-09-08 at 10 27 42" src="https://user-images.githubusercontent.com/37421564/189074356-93e4caa0-b67d-4f8b-b1b7-94b5fc5dd4c8.png">